README.md

```diff
@@ -160,6 +160,13 @@ Horizontal scrollable range dates allow you to set your preferred scrollable ran
 
 ## Installation
 
+### Swift Package Manager
+JZCalendarWeekView can be added to your project by adding the following dependency to your `Package.swift`:
+
+```swift
+.package(url: "https://github.com/Alamofire/Alamofire.git", .upToNextMajor(from: "0.7.2"))
```

Comment:
Can we change this one to JZCalendarWeekView

Reply:
oops copy pasta 😄

Reply:
One more thing, should this be 0.7.1?

Reply:
well the 0.7.1 tag does not have the `Package.swift` file. This is assuming there will be a 0.7.2 release enabling swift pm